core/src/main/java/greencity/controller/HabitAssignController.java

```diff
@@ -150,27 +150,27 @@ public ResponseEntity<List<HabitAssignDto>> getCurrentUserHabitAssignsByIdAndAcq
     /**
      * Method that return UserShoppingList and CustomShoppingList.
      *
-     * @param habitId {@link HabitVO} id.
-     * @param userVO  {@link UserVO} instance.
-     * @param locale  needed language code.
+     * @param habitAssignId {@link HabitAssignVO} id.
+     * @param userVO        {@link UserVO} instance.
+     * @param locale        needed language code.
      * @return User Shopping List and Custom Shopping List.
      */
-    @ApiOperation(value = "Get user shopping and user custom shopping lists")
+    @ApiOperation(value = "Get user shopping and custom shopping lists")
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = HttpStatuses.OK,
             response = UserShoppingAndCustomShoppingListsDto.class),
         @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
         @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED)
```

Comment:
cam we have `404` response?
For example, if there are no record in the database with `habitAssignId` or this `habitAssignId` does not belong to current user